### PR TITLE
rotating small coords lines on camera postion

### PR DESCRIPTION
**Pull request by alonss0**
#861mbmeq2

### CODE REVIEW

**Comment by cblech** [comment]
The handles should not rotate freely to face the camera. Instead, each should be in one of four states. Take inspiration from how it is done in old dlc-edit

https://user-images.githubusercontent.com/11379989/222638918-34b07711-50d0-4f08-95a7-f317d4d12a18.mp4

File: Assets/Scripts/Visuals/GizmoSizeVisuals.cs [hunk 1]
```diff
@@ -8,11 +9,14 @@ public class GizmoSizeVisuals : MonoBehaviour
     {
         // Dependencies
         private GizmoSizeSystem gizmoSizeSystem;
+        private CameraState cameraState;
```
Comment (cblech) [inline Assets/Scripts/Visuals/GizmoSizeVisuals.cs]: Is this needed?

**Review by cblech — CHANGES_REQUESTED** [review]

File: Assets/Scripts/Visuals/GizmoSizeVisuals.cs (hunk 1 above)
Comment (alonss0) [reply Assets/Scripts/Visuals/GizmoSizeVisuals.cs]: I use it to obtain camera position values

Comment (cblech) [reply Assets/Scripts/Visuals/GizmoSizeVisuals.cs]: But you are not using it in this file. Only in the LineRotator. And there you are injecting it separately [(here)](https://github.com/metagamehub/dcl-edit/pull/148/files#diff-3b8d109d316abe060bcf0fc6892f8298ddd7affbfec394174038cf91fad82075R24)

Comment (alonss0) [reply Assets/Scripts/Visuals/GizmoSizeVisuals.cs]: Oh yes, I'm sorry, I'm going to fix it.

**Review by cblech — APPROVED** [review]

**Pull request merged**